lib/gerry/client/access.rb

```diff
@@ -0,0 +1,17 @@
+require 'erb'
+
+module Gerry
+  class Client
+    module Access
+      # Get access rights for the specified project
+      #
+      # @param [VarArgs] projects the project names
+      # @return [Hash] the list of access rights
+      def access(*projects)
+        projects = projects.flatten.map { |name| ERB::Util.url_encode(name) }
```

Comment:
I see what you mean, but if I'm not mistaken you're not using `URI.escape()` yet either. In fact, I was using that at first, only to see that it did not properly escape slashes (`/`) in nested project names like `raytracer/math` to `raytracer%2Fmath`. On the other hand, it also seemed to work without the escaping, but I really think the slash *should* be escaped here, or?